ui-ngx/src/app/modules/home/components/dashboard-page/dashboard-page.component.html

```diff
@@ -74,8 +74,8 @@
               {{'dashboard.states-short' | translate}}
             </button>
         </ng-container>
-          <img *ngIf="showDashboardLogo()" [src]="dashboardLogo | image | async"
-               aria-label="dashboard_logo" class="dashboard_logo"/>
+          <img *ngIf="showDashboardLogo()" [src]="dashboardLogo | image | async" (click)="goHome()"
+               aria-label="dashboard_logo" class="dashboard_logo pointer-events-auto cursor-pointer"/>
```

Comment:
It might be better to add the ability to open via right mouse button or long tap